yellowbrick/features/pca.py

```diff
@@ -177,7 +178,7 @@ def draw(self, **kwargs):
             self.fig = plt.figure()
             self.ax = self.fig.add_subplot(111, projection='3d')
             self.ax.scatter(X[:, 0], X[:, 1], X[:, 2],
-                            c=self.color, cmap=self.colormap)
+                            c=self.color, cmap=self.colormap,alpha=self.alpha)
```

Comment:
place a space after the comma right before alpha

Reply:
OK. Will do so!